shared/js/background/utils.es6.js

```diff
@@ -150,14 +150,16 @@ function getAsyncBlockingSupport () {
  * check to see if this is a broken site reported on github
 */
 function isBroken (url) {
-    if (!tdsStorage?.brokenSiteList) return
+    if (!tdsStorage || !tdsStorage.brokenSiteList) return
     return isBrokenList(url, tdsStorage.brokenSiteList)
 }
 
 function isCanvasBroken (url) {
-    if (!tdsStorage?.fingerprinting?.canvas?.sites) return
+    if (!tdsStorage || !tdsStorage.fingerprinting || !tdsStorage.fingerprinting.canvas) return
```

Comment:
It seems like these changes here came in with 19bd1fe7b67da32c1df051100ecabc98bddedd32 can we remove them? Or were they intended?

Reply:
@SlayterDev did you have some issue with optional chaining that prompted you to change this?

@jonathanKingston I assume this gets transpiled as we target a firefox min version of 57, which is before support for this was added https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
Yeah this this super easy to read statement:
```
return (tdsStorage === null || tdsStorage === void 0 ? void 0 : (_tdsStorage$fingerpri2 = tdsStorage.fingerprinting) === null || _tdsStorage$fingerpri2 === void 0 ? void 0 : (_tdsStorage$fingerpri3 = _tdsStorage$fingerpri2.canvas) === null || _tdsStorage$fingerpri3 === void 0 ? void 0 : _tdsStorage$fingerpri3.scripts) || [];
```

Reply:
Yes I made this change. I was having issues with linting with the optional chain and @kdzwinel suggested we refactor it as optional chaining isn't well supported in the linter yet.

Reply:
The linter should support this now after updating the the latest "standard"

Reply:
Could've been. I have no issue switching it back as the optionals were much cleaner. I'll give it a shot.